extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -47,6 +47,13 @@
     @ConfigItem(defaultValue = "8444")
     public int testSslPort;
 
+    /**
+     * Whether the HTTP port should be available for use
+     * if not any requests to the HTTP port will be redirected to the HTTPS port
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean portEnabled;
```

Comment:
I don't think this is a very descriptive config property, both http and https are configured here so it is not clear what this does.. Maybe 'requireHttps' would be a better name.